packages/matter-node.js/tsconfig.base.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "target": "ES2015",
     "module": "CommonJS",
+    "strictNullChecks": true
```

Comment:
Add a comment what this is needed and what is the effect

Reply:
added  // Required to correctly generate Nullable types in d.ts